spec/factories/submission.rb

```diff
@@ -2,5 +2,12 @@
   factory :submission do
     submission_type :population
     user
+
+    trait :finalized do
```

Comment:
I introduced such trait too in #178 but it is more havy-weight  as it assigns `submission.content` and performs finalization. I think it is worth to keep it but maybe under different name.